lib/twitter/media/photo.rb

```diff
@@ -12,10 +12,11 @@ class Photo < Twitter::Identity
       #
       # @return [Array<Twitter::Size>]
       def sizes
-        @attrs.fetch(:sizes, []).inject({}) do |object, (key, value)|
+        object = {}
+        @attrs.fetch(:sizes, []).each do |(key, value)|
           object[key] = Size.new(value)
-          object
         end
+        object
```

Comment:
@sferik Originally,  #inject was there in [5-stable](https://github.com/sferik/twitter/blob/5-stable/lib/twitter/media/photo.rb) branch. While backporting media entities, those files are replaced with new files from v6 branch.